canonical-web-and-design/react-components
# Prepare 0.37.3

## Done

- Prepare docs for the release 0.37.3

## QA

- Full code review not needed (code auto generated) - only check if version in package.json is correct
- Make sure demo runs correctly
